code/src/index.css

```diff
@@ -1,13 +1,245 @@
 body {
```

Comment:
sticking this note at the top of this document and then will add notes for each occurence below: a gengle reminder to make sure you go through and clean your code and adjust for consistent spacing, etc. 😊

Reply:
Me, spaces and indentation.. always a battle 🥴, I did correct all of the lines below, but let me know if it's okay!